urbanroute/urbanroute/geospatial/ellipses.py

```diff
@@ -0,0 +1,46 @@
+import matplotlib.pyplot as plt
+import math
+from shapely.geometry import Point, Polygon
+import shapely.affinity
```

Comment:
Is shapely in the urbanroute setup.py file? Or is it already included due to geopandas & osmnx?